boards/BMS/Src/App/App_Accumulator.c

```diff
@@ -20,6 +20,20 @@ enum AccumulatorMonitorState
     GET_CELL_TEMP_STATE,
 };
 
+struct VoltageStats
+{
+    struct
+    {
+        uint8_t  segment;
+        uint8_t  series_elem;
+        uint16_t voltage;
+    } min, max;
```

Comment:
super nit: I'd add a line here for readability 
```suggestion
    } min, max;
    
```